empress/__init__.py

```diff
@@ -95,6 +95,25 @@ def draw_on(self, axes: plt.Axes):
         recon_viewer.render(self.recon_input.host_dict, self.recon_input.parasite_dict, self._reconciliation,
                             axes=axes)
 
+    def count_events(self) -> Tuple[int, int, int]:
```

Comment:
We should also count (and, ultimately report) the number of cospeciations.  The truth is that we can infer the number of cospeciations from dup_count, trans_count, and loss_count, but we may as well compute them here just as we do with other events.

Reply:
ok